lib/coap/socket_server.ex

```diff
@@ -46,10 +46,20 @@ defmodule CoAP.SocketServer do
     Open a udp socket on the given port and store in state
     Initialize connections and monitors empty maps in state
   """
-  def init([endpoint, port]) do
+  def init([endpoint, port]), do: init([endpoint, port, []])
+
+  def init([endpoint, port, config]) when is_list(config) do
```

Comment:
I suppose I could get rid of this `is_list` check, in the off-chance the `config.exs` uses a `%{}`.